samples/ClassroomLabs/Modules/Library/Tests/Lab.tests.ps1

```diff
@@ -0,0 +1,74 @@
+[cmdletbinding()]
+Param()
+Import-Module $PSScriptRoot\..\Az.LabServices.psm1
+
+$rgName = 'TestPerm'
```

Comment:
Is there any way we can prepend the resource names (at least RGs and Lab Accounts) with something across all the tests?  That way I know what resources in the subscription are connected to the automation?  On the other library I put "DtlLibrary…" at the beginning...

Reply:
Done. AzLabsLibrary is the prefix.